src/org/mockito/verification/VerificationMode.java

```diff
@@ -33,4 +33,11 @@
     
     void verify(VerificationData data);
     
+    /**
+     * Description will be prepended to the assertion error if verification fails.
+     * @param description The custom failure message
+     * @return VerificationMode
+     * @since 2.0.3
+     */
+    VerificationMode description(String description);
```

Comment:
I think a separate interface would be better. I'm leaning to something like `Descriptable`

But maybe a sub-interface like `DescriptableVerificationMode` could do it too.